src/Boolean.php

```diff
@@ -61,7 +61,7 @@ class Boolean extends AbstractFilter
      */
     public function __construct($typeOrOptions = null, $casting = true, $translations = [])
     {
-        if ($typeOrOptions !== null) {
+        if ($typeOrOptions !== null && $typeOrOptions !== []) {
```

Comment:
I think a  `empty($typeOrOptions)` condition dose the same, but is easier to read.